index.d.ts

```diff
@@ -0,0 +1,4 @@
+export default class RemoteInputElement extends HTMLElement {
+  readonly input?: HTMLInputElement;
```

Comment:
What does `input?` do here? We seem to do this elsewhere:

```suggestion
  readonly input: HTMLInputElement | undefined;
```

And the flow def is actually incorrect in that we should accept `HTMLTextAreaElement` too:

https://github.com/github/remote-input-element/blob/2b899d0044671e9f17e1ec45569dd1cf152584d2/index.js#L41-L44

Reply:
Sorry this was due to me having a really old VSCode that had outdated TypeScript installed.